tests/templates/kuttl/oidc/35_authentication-class.yaml.j2

```diff
@@ -7,7 +7,7 @@ spec:
   provider:
     oidc:
       hostname: keycloak.$NAMESPACE.svc.cluster.local
-      rootPath: /realms/test
+      rootPath: /realms/test/
```

Comment:
Thanks for updating the integration test, will do so for my PRs as well!
Can you please also add a Rust test similar to what the trino and nifi PRs are doing?